INSTALL.md

```diff
@@ -12,6 +12,7 @@ required:
 * pkg-config
 * glib and gio 2.0 libraries and development files
 * libtss2-sys, libtss2-mu and TCTI libraries from https://github.com/tpm2-software/tpm2-tss
+* dbus
```

Comment:
This is a runtime dependency required for integration test.
Should be listed below with the other test suite dependencies.

Reply:
It's more than just that it comes with dbus-run-session, you need dbus for the whole IPC mechanism. Without dbus it won't work right? So you need dbus for the IPC daemon.

```bash
apt show dbus
<snip>
This package contains the D-Bus daemon and related utilities.
<snip>
```

Reply:
yes, you're right